dnd/app.js

```diff
@@ -0,0 +1,57 @@
+const express = require('express')
+const app = express()
+const morgan = require('morgan')
+const bodyParser = require ('body-parser')
+const mongoose = require ('mongoose')
+
+const encounterRoutes = require('./api/routes/encounters')
+const characterRoutes = require('./api/routes/characters')
+const initiativeRoutes = require('./api/routes/initiatives')
+
+mongoose.connect(
+  'mongodb://localhost:27017/dndb',
```

Comment:
Make use of ENV variables. Also it would be a good idea to try connect to mongo before starting the sever. Probably in `server.js`

Reply:
Also let´s add event handlers to mongoose validate the connection, i.e
```
mongoose.connection
.once('open', () => /* Do something*/  )
.on('error', ( err) => /* Handle Error */)
```